src/svg-icons-builder/index.ts

```diff
@@ -1,35 +1,53 @@
-import { BuilderOutput, createBuilder } from '@angular-devkit/architect';
+import { BuilderContext, BuilderOutput, createBuilder } from '@angular-devkit/architect';
 import { JsonObject } from '@angular-devkit/core';
-import { fork } from 'child_process';
-import { join } from 'path';
-import { ConstantsConversionOptions } from 'svg-to-ts/src/lib/options/conversion-options';
+/*
+ TODO - we should export this from svg-to-ts - otherwise we rely on internals and it would break if
+ we refactor something in svg-to-ts
+ */
+import {
+  CommonConversionOptions,
+  ConstantsConversionOptions,
+  ConversionType,
+  FileConversionOptions,
+  ObjectConversionOptions,
+} from 'svg-to-ts/src/lib/options/conversion-options';
+import { convertToFiles } from 'svg-to-ts/src/lib/converters/files.converter';
+import { convertToConstants } from 'svg-to-ts/src/lib/converters/constants.converter';
+import { convertToSingleObject } from 'svg-to-ts/src/lib/converters/object.converter';
 
-interface Options extends ConstantsConversionOptions, JsonObject {
+interface Options extends CommonConversionOptions, JsonObject {
   generateCompleteIconSet: boolean; // TODO: should this be exportCompleteIconSet
 }
 
-export default createBuilder<Options>((options, context) => {
-  context.logger.info(`Options: ${options}`);
-  return new Promise<BuilderOutput>((resolve, reject) => {
-    const child = fork(join(__dirname, `/svg-to-ts-wrapper.ts`), [], {
-      stdio: [0, 1, 2, 'ipc'],
-    });
-    child.send(options);
-    child.on('message', ({ isError, data }) => {
-      if (isError) {
-        context.logger.error(data);
-        child.kill('SIGINT');
+export default createBuilder<Options>((conversionOptions: Options, context: BuilderContext) => {
```

Comment:
I think this is simpler than my original approach, I did figure out how to make it work but requires always using the ` dist` folder.

Reply:
Yes, I also think it's simpler and it also fixes an error I got when running it in a real application.